syscall.go

```diff
@@ -5,6 +5,13 @@
 
 package purego
 
+// CDecl marks a function as being called using the __cdecl calling convention as defined in
+// the [MSDocs] when passed to NewCallback. It must be the first argument to the function.
+// This is only useful on 386, but it is safe to use on other platforms.
```

Comment:
Is this only for Windows, or also for the other OSes?

Reply:
It's visible on all OSs so that there is no need to write basically the same thing on each

Reply:
So

> This is only useful on 386

This is only useful on 386 Windows, right?

Reply:
Correct

Reply:
Let's make this explicit in this comment. (Just s/386/386 Windows/ is fine to me)